leap.json

```diff
@@ -26,7 +26,7 @@
          "expected": true
       },
       {
-         "description": "Y2K",
+         "description": "y2k",
```

Comment:
Let's just let each track downcase if they want. In Go it's common to use `CamelCase`, in Ruby, `snake_case`. I don't know about other languages.